src/lexer.l

```diff
@@ -1,7 +1,7 @@
 %{
 #include <assert.h>
-#include "jv_alloc.h"
-#include "compile.h"
+#include "src/jv_alloc.h"
```

Comment:
Oh?  `-I${srcdir}` is not enough?

Reply:
I ran make with `V=1` and sure enough, there were just `-I. -I..`. The `..` is the `-I${srcdir}`, not `root/src`.

Reply:
I see the same.  Why does jq build for me on Unix when building out of tree?